unlock-app/src/hooks/useLock.tsx

```diff
@@ -355,6 +355,7 @@ export const useLock = (lockFromProps: Partial<Lock>, network: number) => {
   const config = useConfig()
   const [error, setError] = useState<string | null>(null)
 
+  // TODO: to remove? not used anywhere
   const getLock = async (opts: any = {}) => {
```

Comment:
can we remove? not used anywhere in app, will simplify also `useLock` hook

Reply:
I think so

Reply:
ill do it in a different PR after i merge this one